lib/controllers/page_controller.rb

```diff
@@ -30,13 +30,16 @@ def restrict_concealed(page)
 
   get '/list' do
     @page_title = "Innehållsförteckning"
-    @page_groups = Page.order(:title_char, :title).to_hash_groups(:title_char)
+    @page_groups = Page.order(:title_char, :title).
+      with_concealed_if(starkast?).
+      to_hash_groups(:title_char)
     haml :index
   end
 
   get '/latest' do
     @page_title = "Senast ändrad"
     @page_groups = Page.order(:updated_on).reverse.
+      with_concealed_if(starkast?).
```

Comment:
How about we change to dot first style @dentarg ?

Reply:
Sure

Reply:
Fixed in da5a0b9d9d4244b9127ec2826b246bc96ffa7f2c